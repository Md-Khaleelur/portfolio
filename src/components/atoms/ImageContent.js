import React from 'react';
import "../template/styles.css";

export default function ImageContent() {
    
    return ( 
            <div>
                <img src={this.props.sourceName} alt="img-miss" />
            </div>
        
    )

}
