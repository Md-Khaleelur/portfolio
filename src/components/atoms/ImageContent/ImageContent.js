import React, { Component } from 'react';


export default class ImageContent extends Component {
    render() {
        return (
            <div          
                data-testid="imageContent"
            >
                <img src={this.props.sourceName} alt="img-miss" />
            </div>
        )
    }
}
