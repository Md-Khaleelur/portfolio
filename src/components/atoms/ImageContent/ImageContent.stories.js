import React from 'react';
import ImageContent from './ImageContent';
import me2 from '../../../img/me2.png';

export default { 
    title: 'ImageContent',

};

export const withImg = () => <ImageContent sourceName={me2}/>
