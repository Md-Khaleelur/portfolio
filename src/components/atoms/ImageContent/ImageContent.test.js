import React from 'react';
import { create } from "react-test-renderer";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ImageContent from './ImageContent';

describe("ImageContent component",() =>{
    test("Matches the snapshot",() => {
        const imageContent = create(<ImageContent/>);
        expect(imageContent.toJSON()).toMatchSnapshot();

    })
    test("ImageContent is rendered",() => {
        const { getByTestId, getByText } = render(<ImageContent/>);
        expect(getByTestId("imageContent")).toBeInTheDocument();
        
    });
   

});