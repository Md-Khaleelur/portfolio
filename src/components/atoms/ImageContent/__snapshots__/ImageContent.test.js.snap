// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageContent component Matches the snapshot 1`] = `
<div
  data-testid="imageContent"
>
  <img
    alt="img-miss"
  />
</div>
`;
